omero/developers/json-api.rst

```diff
@@ -210,7 +210,7 @@ It uses the python ``requests`` library to connect to the JSON api, login, query
 These steps are covered in more detail below.
 
 For an example how to use the API with Java,
-see :omedocs:`blob/develop/omero/examples/java/JSONClient.java`.
+see :omedocs:`JSONClient.java <blob/develop/omero/examples/java/JSONClient.java>`.
```

Comment:
@sbesson I think this will work fine when merged because it works locally, but it's making the merge build yellow because the path isn't right for snoppy's branch - is there any way to fix this other than hardcoding the link instead?